lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -157,12 +157,6 @@
         % endif
       % endif
 
-      % if cert_status['status'] == 'downloadable' and enrollment.mode == 'verified' and cert_status['mode'] == 'honor':
-        <div class="certificate-explanation">
-            ${_('Since we did not have a valid set of verification photos from you when your {cert_name_long} was generated, we could not grant you a verified {cert_name_short}. An honor code {cert_name_short} has been granted instead.').format(cert_name_short=cert_name_short, cert_name_long=cert_name_long)}
```

Comment:
I didn't see a use case for this message, because a cert that isn't verified will have an `unverified` status as opposed to downloadable. I also don't think this template is being used anywhere.